src/core/ldap/domain/ldap-event-handler.ts

```diff
@@ -152,17 +153,54 @@ export class LdapEventHandler {
             `Received PersonenkontextUpdatedEvent, personId:${event.person.id}, new personenkontexte: ${event.newKontexte.length}, deleted personenkontexte: ${event.removedKontexte.length}`,
         );
 
+        if (!event.containsAnyCurrentPKWithRollenartLehr()) {
+            // Delete all removed personenkontexte if rollenart === LEHR
+            await Promise.allSettled(
```

Comment:
An dieser Stelle werden die Fehler verschluckt.

Reply:
Soll über die Results iteriert werden oder soll der Fehler direkt in der map Funktion behandelt werden?

Reply:
Ist beides OK. Aber irgendwo müssen sie geloggt werden.